packages/editor/src/components/block-vertical-alignment-toolbar/README.md

```diff
@@ -0,0 +1,84 @@
+BlockVerticalAlignmentToolbar
+=============================
+
+`BlockVerticalAlignmentToolbar` is a simple `Toolbar` component designed to provide _vertical_ alignment UI controls for use within the editor `BlockControls` toolbar.
+
+This builds upon similar patterns to the [`BlockAlignmentToolbar`](https://github.com/WordPress/gutenberg/tree/master/packages/editor/src/components/block-alignment-toolbar) but is focused on vertical alignment only.
+
+## Usage
+
+In a block's `edit` implementation, render a `<BlockControls />` component. Then inside of this add the `<BlockVerticalAlignmentToolbar />` where required. 
+
+
+```jsx
+import { registerBlockType } from '@wordpress/blocks';
+import { Fragment } from '@wordpress/element';
+import {
+	BlockControls,
+	BlockVerticalAlignmentToolbar,
+} from '@wordpress/editor';
+
+registerBlockType( 'my-plugin/my-block', {
+	// ...
+
+	attributes: {
+		// other attributes here
+		// ...
+
+		verticalAlignment: {
+			type: 'string',
+		},
+	},
+
+	edit( { attributes, setAttributes } ) {
+		
+		const { verticalAlignment } = attributes;
+
+		// Change handler to set Block `attributes`
+		const onChange = ( alignment ) => setAttributes( { verticalAlignment: alignment } );
+
+		return (
+			<Fragment>
+				<BlockControls>
+					<BlockVerticalAlignmentToolbar
+						onChange={ onChange }
+						value={ verticalAlignment }
+					/>
+				</BlockControls>
+				<div className={ className }>
```

Comment:
Should `className` variable be removed from the example since it is never defined? It might be confusing for those who copy & paste examples.

Reply:
👍